src/get-started/editor.md

```diff
@@ -30,9 +42,6 @@ that's OK, skip ahead to the [next step: Test drive][].
   <li class="nav-item">
     <a class="nav-link" id="androidstudio-tab" href="#androidstudio" role="tab" aria-controls="androidstudio" aria-selected="false">Android Studio and IntelliJ</a>
   </li>
-  <li class="nav-item">
```

Comment:
**comment:** Yes. We only included it in the editor page, but not in the test drive page or potentially other pages. If we list it in a place of prominence, it should retain that prominence throughout. As it didn't, it made more sense to concentrate on our main IDEs and let Emacs fall into the "everything else" category.

Reply:
I think it's fine if we lower the prominence, but it's unfortunate to lose the mention and link altogether since the plugin looks maintained. It might discourage emacs users or at least slow their getting started experience down. Could we somehow keep a brief mention or link?